pkg/parser/node.go

```diff
@@ -103,6 +107,25 @@ func (n *Node) validate(pctx *UnixParserCtx, ectx EnricherCtx) error {
 			}
 		}
 	}
+
+	for idx, stash := range n.Stash {
```

Comment:
We probably want to only process the stash if the grok pattern succeeded ?

Reply:
Asked the same question, but we might want to capture some date based on a grok that is done at a higher level :


```yaml
onsuccess: next_stage
debug: true
filter: "evt.Parsed.program == 'test'"
name: crowdsecurity/test-logs
description: "Parse test logs"
grok:
  pattern: something something capture %{DATA:event _type}
nodes:
  -  filter: evt.Parsed.event_type == "case1"
     static:
      ...
  - filter: evt.Parsed.event_type == "case2"
    stash:
      - name: ...

```